hazelcast/src/main/java/com/hazelcast/map/impl/MapClientAwareService.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.map.impl;
+
+import com.hazelcast.spi.ClientAwareService;
+
+/**
+ * Defines {@link ClientAwareService} behavior of map service.
+ *
+ * @see ClientAwareService
+ */
+class MapClientAwareService implements ClientAwareService {
```

Comment:
Ok. So you override this structure in enterprise by something sensible. 

I'm fine with adding a No-implementation. Can you add it to the documentation + Can you remove the constructor and the field since it isn't used?